ts/services/audioRecorder.ts

```diff
@@ -12,6 +14,18 @@ export class RecorderClass {
   private blob?: Blob;
   private resolve?: (blob: Blob) => void;
 
+  private async requestMicrophonePermissions(): Promise<boolean> {
```

Comment:
Can you please make this function static or pull it out altogether so we don't have to eslint-disable class-methods-use-this?